script/electron-builder-linux.yml

```diff
@@ -7,5 +7,14 @@ linux:
   target:
     - deb
     - rpm
+    - snap
     - AppImage
   maintainer: 'GitHub, Inc <opensource+desktop@github.com>'
+snap:
+  confinement: 'classic'
+  stagePackages:
+    - default
+    - git-core
+    - libcurl3-gnutls
+    - libsecret-1-0
+    - openssh-client
```

Comment:
I gather this is related to connecting to SSH remotes? Thanks for the reminder!

Reply:
Yes, when I was testing GitHub Desktop as a strictly confined Snap I discovered that it was attempting to execute some of the SSH client utilities, so I bundled `openssh-client` to satisfy the requirement.

Now that the GitHub Desktop Snap is using classic confinement it will be able to execute ssh client utilities that are installed on the host. It's your choice if you want to bundle them in the Snap or not, let me know how you'd like to proceed.

Reply:
I'm fine with leaving this as-is, I just need to ensure the other packaging dependencies are consistent with this approach.